icu4c/source/i18n/number_decimalquantity.cpp

```diff
@@ -1323,8 +1326,8 @@ bool DecimalQuantity::operator==(const DecimalQuantity& other) const {
     }
 }
 
-UnicodeString DecimalQuantity::toString() const {
-    MaybeStackArray<char, 30> digits(precision + 1);
+UnicodeString DecimalQuantity::toString(UErrorCode& status) const {
+    MaybeStackArray<char, 30> digits(precision + 1, status);
```

Comment:
Optional: remove the `UErrorCode&` parameter that you added to the function, and then write

```cpp
UErrorCode localStatus = U_ZERO_ERROR;
MaybeStackArray<char, 30> digits(precision + 1, localStatus);
if (U_FAILURE(localStatus)) {
    return ICU_Utility::makeBogusString();
}
```

Reply:
Done.